conf/deployment/vsphere/ipi_1az_rhcos_vmfs_3m_3w.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# This config is suppose to work on most of DCs we have.
```

Comment:
Note Must be changed to `This config is suppose to work on DC-CP, DC-ECO, DC-PS`

Reply:
Done